cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -68,3 +73,16 @@ func ValidateTokenDiscovery(c *kubeadm.TokenDiscovery, fldPath *field.Path) fiel
 	allErrs := field.ErrorList{}
 	return allErrs
 }
+
+func ValidateServiceSubnet(subnet string, fldPath *field.Path) field.ErrorList {
```

Comment:
That's right, but since there are no tests at all for this package, can we do it in a follow-up?
I'd like to get this merged asap so I can continue with the next PR

Sounds good?